package.json

```diff
@@ -31,7 +31,8 @@
     "babel-plugin-transform-async-to-generator": "^6.24.1",
     "body-parser": "^1.19.0",
     "co": "^4.6.0",
-    "ember-cli-babel": "^7.11.0",
+    "ember-cli-babel": "^7.17.2",
+    "ember-cli-htmlbars": "^4.2.2",
```

Comment:
Can you move this back into devDependencies? I do not believe that it is needed as a dep.

Reply:
Done